docs/hmrc.html

```diff
@@ -833,6 +834,14 @@ <h3><a name="msg_head">Message page headers/footers</a></h3>
 <br><i>show_headers = From,Subject,Date,Message-ID</i> (disabled by default)
 </dd>
 
+<a name="show_received_date"></a>
+<dt><strong>show_received_date = [ 0 | 1 ]</strong></dt><dd>
```

Comment:
Please add (enabled by default)  comment.

Reply:
I updated the example and added the comment, as is done for other options that are enabled by default.